topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -9,25 +9,31 @@ questions:
   - How to improve the quality of a sequence dataset?
 objectives:
   - Manipulate FastQ files
-  - Control quality from a FastQ file
+  - Assess quality from a FastQ file
   - Use FastQC tool
   - Understand FastQC output
   - Use tools for quality correction
+  - Use a tool to aggregate FastQC output
+  - Process paired-end data
 requirements:
-time_estimation: "1h"
+time_estimation: "1H"
 key_points:
   - Run quality control on every dataset before running any other bioinformatics analysis
   - Take care of the parameters used to improve the sequence quality
   - Re-run FastQC to check the impact of the quality control
+  - Analyze the forward and reverse reads together
 contributors:
   - bebatut
 ---
 
 # Introduction
 {:.no_toc}
 
-During sequencing, errors are introduced, such as incorrect nucleotides being called. These are due to the technical limitations of each sequencing platform. Sequencing errors might bias the analysis, ultimately leading to a misinterpretation of the data. 
+During sequencing, the sequencer is sequencing each fragment of the library and generating a short sequence, also called a **read**: a succession of nucleotides.
```

Comment:
This sentence has "sequenc*" used a lot (4 times), what about instead something like "During sequencing, the nucleotide bases in a DNA or RNA sample (library) are determined by the sequencer.  For each fragment in the library, a short sequence is generated, also called a **read**, which is simply a succession of nucleotides."